cave/utils/csv2rh.py

```diff
@@ -0,0 +1,250 @@
+import os
```

Comment:
Shouldn't this file be part of cave/reader?
SMAC2 and SMAC3 readers are located there

Reply:
Yes.